test/OpenTelemetry.Exporter.Zipkin.Tests/ZipkinActivityExporterTests.cs

```diff
@@ -127,6 +129,47 @@ public async Task ZipkinActivityExporterIntegrationTest(bool useShortTraceIds)
                 Responses[requestId]);
         }
 
+        [Fact]
+        public void UseZipkinActivityExporterWithCustomActivityProcessor()
+        {
+            const string ActivitySourceName = "zipkin.test";
+            Guid requestId = Guid.NewGuid();
+            TestActivityProcessor testActivityProcessor = new TestActivityProcessor();
+
+            bool startCalled = false;
+            bool endCalled = false;
+
+            testActivityProcessor.StartAction =
+                (a) =>
+                {
+                    startCalled = true;
+                    a.AddTag("zipkinCustomTag", "Custom Tag Value for zipkin");
```

Comment:
startCalled alone is sufficient to validate this was called. Dont think validating the tag is adding extra value to the test.

Reply:
Fixed.